wp-includes/class-wp-https-detection.php

```diff
@@ -57,6 +63,10 @@ class WP_HTTPS_Detection {
 	 * Initializes the object.
 	 */
 	public function init() {
+		if ( apply_filters( self::FILTER_DISABLE, false ) ) {
```

Comment:
Core uses string literals in calls to `apply_filters()` and `do_action()` to facilitate generating documentation. And IDEs like PhpStorm parse strings in code indexing which allow for autocompletion among other things.

```suggestion
		/**
		 * Filters whether the HTTPS Detection and UI should be disabled.
		 *
		 * @param bool $disabled Disabled.
		 */
		if ( apply_filters( 'wp_https_detection_ui_disabled', false ) ) {
```

Reply:
Fair point. I hate repeating strings, but it makes sense for Core.